openapi/pex-openapi.yaml

```diff
@@ -721,6 +760,30 @@ components:
         - proof_type
       additionalProperties: false
 
+    di_object:
+      type: object
+      description: Which type of DataIntegrityProof type is supported
+      properties:
+        proof_type:
+          type: array
+          description: The Proof type DataIntegrityProof
+          minItems: 1
+          maxItems: 1
+          items:
+            type: string
+            description: The Proof type (DataIntegrityProof)
```

Comment:
DI allows other types than `DataIntegrityProof`. So I think `DataIntegrityProof` should be assumed to be supported if at least one `cryptosuite` is defined (it should be optional, or it should be allowed to pass an empty array).